packages/relay/src/lib/types/events.ts

```diff
@@ -0,0 +1,45 @@
+/* -
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+export interface IExecuteTransactionEventPayload {
+  transactionId: string;
+  callerName: string;
+  requestId: string;
+  txConstructorName: string;
+  operatorAccountId: string;
+  interactingEntity: string;
+}
+
+export interface IExecuteQueryEventPayload {
+  executionType: string;
+  transactionId: string;
+  txConstructorName: string;
+  callerName: string;
+  cost: number;
+  gasUsed: number;
+  interactingEntity: string;
+  requestId: string;
+}
+
+export enum ExecutionType {
```

Comment:
Please, change the keys and values of the string enum to be exactly the same. 

As it is right now it is possible that unexpected things happen when parsing (it has happened to me in the past), and we don’t really need them to be different, so let’s make our lives easier.

Reply:
fair! Updated with the fix.

Reply:
Actually, do we need these strings at all? I saw that we use them only for this log:

```
    this.logger.trace(
      `${formattedRequestId} Capturing HBAR charged: executionType=${executionType} transactionId=${transactionId}, txConstructorName=${transactionType}, callerName=${callerName}, cost=${cost} tinybars`,
    );
```

In my opinion, `txConstructorName` provides enough information about the kind of the transaction, why do we need this `executionType` string? It seems useless to me.

Reply:
https://github.com/hashgraph/hedera-json-rpc-relay/pull/2873#discussion_r1738907538